README.md

```diff
@@ -136,7 +136,7 @@ the only CodeBuild Run input you need to provide is the project name.
     aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
     aws-region: us-east-2
 - name: Run CodeBuild
-  uses: aws-actions/aws-codebuild-run-project@v1
+  uses: aws-actions/aws-codebuild-run-build@v1.0.3
```

Comment:
If you want to use `v1` instead of `v1.0.3`, you need to create a tag with the name `v1`.
https://github.com/aws-actions/aws-codebuild-run-build/tags

Reply:
Yup, that is correct. Thanks for the catch.